ddtrace/internal/utils/cache.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, maxsize=256):
         self.lock = RLock()
 
     def get(self, key, f):  # type: ignore[override]
-        # type: (T, F) -> S
+        # type: (T, Callable[[T], S]) -> S
```

Comment:
I think this was fine 🙁 . What's the issue on this line?

Reply:
I think we can drop the definition of `S` and replace its usages with `Any`

Reply:
Mypy is really enforcing that TypeVars are kept consistent on both sides of the type hint, i.e.
`error: A function returning TypeVar should receive at least one argument containing the same Typevar`
I just expanded out the definition of F